groups/instance/lb.tf

```diff
@@ -0,0 +1,103 @@
+resource "aws_lb" "artifactory" {
+  name               = "${var.environment}-${var.service}-lb"
+  internal           = true
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.alb_security_group.id]
+  subnets            = local.automation_subnet_ids
+
+  enable_deletion_protection = true
+
+  tags = {
+    Name        = "${var.environment}-${var.service}-lb"
+  }
+}
+
+resource "aws_lb_target_group" "front_end_8081" {
+  name     = "${var.environment}-${var.service}-tg-8081"
+  port     = 8081
+  protocol = "HTTP"
+  vpc_id   = data.aws_vpc.placement.id
+
+  health_check {
+    enabled             = true
+    interval            = 30
+    path                = "/"
+    port                = "8081"
+    timeout             = 3
+    healthy_threshold   = 3
+    unhealthy_threshold = 3
+    protocol            = "HTTP"
+    matcher             = "200"
+  }
+}
+
+resource "aws_lb_target_group" "front_end_8082" {
+  name     = "${var.environment}-${var.service}-tg-8082"
+  port     = 8082
+  protocol = "HTTP"
+  vpc_id   = data.aws_vpc.placement.id
+
+  health_check {
+    enabled             = true
+    interval            = 30
+    path                = "/"
+    port                = "8082"
+    timeout             = 3
+    healthy_threshold   = 3
+    unhealthy_threshold = 3
+    protocol            = "HTTP"
+    matcher             = "200"
+  }
+}
+
+resource "aws_lb_listener" "listener_8082" {
+  load_balancer_arn = aws_lb.artifactory.arn
+  port              = "8082"
+  protocol          = "HTTP"
+
+  default_action {
+    type             = "forward"
+    target_group_arn = aws_lb_target_group.front_end_8082.arn
+  }
+}
```

Comment:
This listener isn't required as we won't be accessing the ALB via port 8082, only 80 and 443.